photoeditor/src/main/java/ja/burhanrashid52/photoeditor/ShapeView.java

```diff
@@ -0,0 +1,102 @@
+package ja.burhanrashid52.photoeditor;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.PorterDuff;
+import android.graphics.PorterDuffXfermode;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.view.MotionEvent;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+
+public class ShapeView extends View {
```

Comment:
Need to make package-private. We usually don't expose the internal representation of the view to client.